modules/user-access/pages/user-credentials.adoc

```diff
@@ -72,33 +72,40 @@ These operations include:
 
 == Secrets
 
-Secrets are unique strings that serve as a user's credentials in certain circumstances. A user can have multiple secret strings. Each secret is associated with one user and their role for one graph. If the role is revoked, the secret also becomes invalid.
+Secrets are unique strings that serve as a user’s credentials in certain circumstances. A user can have multiple secret strings. Each secret binds to the user who creates it only and inherits all permissions and privileges from the owner. Starting from 4.1.0, secret is no longer bound to graphs and tokens.
```

Comment:
Add a separate line as "Note " for : As of version 4.1.0, secrets are no longer bound to specific graphs and tokens.